packages/components/src/form-token-field/token-input.tsx

```diff
@@ -26,6 +26,7 @@ export function UnForwardedTokenInput(
 		selectedSuggestionIndex,
 		className,
 		onChange,
+		isComboboxControl,
```

Comment:
We should not introduce properties with such descriptive names, hinting at implementation details.

Assuming that we agree on the code changes to `FormTokenField`, we should look into finding a name that is more semantic for this prop. We should also update docs and add unit tests accordingly.

Reply:
That was just to suggest a point on what needs to happen.